packages/coverage/src/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import { BettererCoverageIssue } from './types';
+
+export const uncoveredFile: BettererCoverageIssue = {
```

Comment:
I think both of these helpers are only used in the `coverage/src/coverage.ts` file so they should probably live there.

Reply:
done